infra/tf/k8s_infra/prometheus.tf

```diff
@@ -295,6 +295,7 @@ resource "helm_release" "prometheus" {
 				KubeSchedulerDown = true
 				CPUThrottlingHigh = true
 				KubeJobNotCompleted = true
+				KubeMemoryOvercommit = true
```

Comment:
@MasterPtato I think we should disable overcommit overall, this is a valid alert. I don't think we should be using overcommit since it can lead to unpredictable behavior. The only services using overcommit are the Rivet services afaik.

Reply:
I mean we should update the resource definitions so we're not using overcommit anywhere. The alert should be left enabled.

Reply:
Isn't it just a matter of modifying the namespace config for a cluster to allocate less memory? What code needs to be changed?

Reply:
lib/bolt/core/src/dep/k8s/gen.rs:313 easy fix

Reply:
I'd check any other services that over allocate if possible by using that query that is set up for the alert

Reply:
https://github.com/rivet-gg/rivet/pull/192/commits/e3e26b545e5238e3f18b850eaf26535d682db7f6